client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
Hey, it looks like this .gitignore file is only in the client folder, and you've pushed node_modules in the server folder, we probably don't want the node_modules pushed to the remote repo - it might be good to add a .gitignore to the server folder as well to prevent this :)

Reply:
Good catch good catch, have just updated it

Reply:
Sweet I think I have done this now

I used this line: git rm -r --cached .

Works fine on my machine but does it still work for you?

Reply:
Yep all works, will merge it :)